k8s/helm/values.yaml

```diff
@@ -23,3 +23,13 @@ kubeRbacProxy:
   resourceRequests:
     cpu: 5m
     memory: 64Mi
+healthConfig:
+  components:
+    required:
+      - name: k8s-etcd
+      - name: k8s-dogu-operator
+      - name: k8s-service-discovery
+      - name: k8s-component-operator
```

Comment:
longhorn is (when installed) critical for the system, too. It should exist some documentation that it needs to be added in valuesYamlOverwrite to be checked aswell.

Reply:
nvm just read the story again. But the docs would be helpful either case, wouldn't they?

Reply:
Added docs